pkgs/net-doc/net/scribblings/ftp.scrbl

```diff
@@ -127,10 +136,15 @@ is intended to limit polling.
                                       #f])
          void?]{
 
-Upload @racket[file-path] to the server's current directory using the same name. 
-If the file already exists in the local directory, it is replaced.
-The @racket[progress-proc] argument is used in the same way as in @racket[ftp-download-file],
-but to report uploaded bytes instead of downloaded bytes.}
+When @racket[port] is @racket[#f], upload @racket[file-path] to the server's
+current directory using the same name.  If the file already exists in the local
+directory, it is replaced.  The @racket[progress-proc] argument is used in the
+same way as in @racket[ftp-download-file], but to report uploaded bytes instead
+of downloaded bytes.
+
+If @racket[port] is an @racket[input-port?], the content of that port is
+streamed as upload to the server and stored under given @racket[file-path]
+name.}
```

Comment:
The port is closed after the content is uploaded, right?

Reply:
It is true, the transfer-data closes the both "to" and "from" ports after it finishes given transfer. This paragraph should probably be amended accordingly.